docs/operator-manual/application.yaml

```diff
@@ -48,6 +48,9 @@ spec:
               hosts:
                 - mydomain.example.com
 
+      # Optional helm version to template with
```

Comment:
Do you think we should specify in the comment line what the default version would be, if nothing is specified
?

Reply:
Same answer as https://github.com/argoproj/argo-cd/pull/4216#discussion_r481259819 above.

Reply:
Fixed in https://github.com/argoproj/argo-cd/commit/3011e10a69f5fa41c45cd329416d0500f1e19fe2